slugid/slugid.py

```diff
@@ -38,6 +43,6 @@ def nice():
     Potentially other "nice" properties may be added in future to further
     restrict the range of potential uuids that may be generated.
     """
-    rawBytes = uuid.uuid4().bytes
-    rawBytes = chr(ord(rawBytes[0]) & 0x7f) + rawBytes[1:]  # Ensure slug starts with [A-Za-f]
+    rawBytes = bytearray(uuid.uuid4().bytes)
+    rawBytes[0] = rawBytes[0] & 0x7f;
```

Comment:
* I think we can lose the trailing `;`
* let's keep the comment to explain what this line does (`# Ensure slug starts with [A-Za-f]`)